### PR TITLE
feat: add ci for linting

### PR DESCRIPTION
**Pull request by kiranjthomas**
@geramirez what do you think about adding some ci to the project? In this PR, I'm proposing adding a github action to lint server/index.js with prettier. I kept accidentally formatting index.js with prettier via vscode and I thought it would be nice to add an opinionated code formatted. Later on, we can add more functional tests.

If you are into this idea, I *think* we will need to enable github actions in the repo settings. I think that's the case since the action didn't already run in this PR. I don't have the permissions to change repo settings at the moment.

### CODE REVIEW

**Comment by geramirez** [comment]
I was thinking about adding some linting. This looks great.

File: .github/workflows/actions.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    branches: [ $default-branch ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
```
Comment (geramirez) [inline .github/workflows/actions.yaml]: We probably only need one node version here.

**Review by geramirez — APPROVED** [review]

File: .github/workflows/actions.yaml (hunk 1 above)
Comment (kiranjthomas) [reply .github/workflows/actions.yaml]: oh good idea. i just enabled 12.x

**Comment by kiranjthomas** [comment]
@geramirez can you enable github actions for the repo? i *think* that is why the linting step is not running.

**Comment by geramirez** [comment]
looks like they should be on: 
image

Let's try merging

**Comment by kiranjthomas** [comment]
the lint job was failing because it couldn't find the `knex` via the @mueller-splash-pad/knex dependency. this bug report seemed relevant: https://npm.community/t/6-8-0-npm-ci-fails-with-local-dependency/5385/10

switching to node 16.4.0 resolved the issue.

@geramirez what do you think?

**Comment by geramirez** [comment]
@kiranjthomas yes that makes sense. I forgot that local packages in older versions of node were linked with npm link

**Pull request merged**